src/tests/Microsoft.PowerFx.Connectors.Tests/ConnectorWizardTests.cs

```diff
@@ -139,7 +139,7 @@ public void ConnectorWizardTest()
             // Finally contruct the expression
             string expression = function.GetExpression("SQL", parameters);
 
-            Assert.Equal(@"SQL.ExecuteProcedureV2(""default"", ""default"", ""sp_2"", { p1: Float(50), p2: ""abc"" })", expression);
+            Assert.Equal(@"SQL.ExecuteProcedureV2(""default"", ""default"", ""sp_2"", { p1: Decimal(50), p2: ""abc"" })", expression);
```

Comment:
@LucGenetier - fyi, this may be a merge conflict for you

Reply:
Nope, seems ok.